lab_1/main.py

```diff
@@ -2,18 +2,35 @@
 Lab 1
 A concordance extraction
 """
+import re
 
+def tokenize(text: str) -> list:
```

Comment:
leave the function description (everywhere)

def tokenize(text) :
     """
     Splits sentences into tokens, converts the tokens into lowercase, removes punctuation
     :param text: the initial text
     :return: a list of lowercased tokens without punctuation
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
   YOUR CODE